.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,12 +1,45 @@
 ---
 name: Feature Request
-about: You want to suggest a new feature for Atmosphère.
-
+about: Suggest a new feature for Atmosphère.
+#assignees:
 ---
 
 ## Feature Request
-
+[ If any section does not apply, use nil. ]  
+[ Lines between [ ] (square brackets) should be removed before posting. ]
 ### What feature are you suggesting?
+#### Overview:
+- [ Include the basic, high-level concepts for this feature here. ]
 
-### Why would this feature be useful?
+#### Smaller Details: 
+- [ These may include specific methods of implementation etc. ]
+
+#### Nature of Request:  
+[ Remove all that do not apply to your request. ]
+- Addition
+  - [ Ex: Addition of certain original features or features from other community projects. ]
+  - [ If you are suggesting porting features or including features from other projects, include what license they are distributed under and what, if any libraries those project use. ]
+- Change
+- Removal
+  - [Ex: Removal of certain features or implementation due to a specific issue/bug or because of low quality code, etc.]
 
+[ Required: Reference the already existing [feature request. ](https://github.com/Atmosphere-NX/Atmosphere/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22features%2Ffeature-request%22) before you make your own request. ]  
```

Comment:
We can phrase this more politely.
```suggestion
[ Note: Please [search for previous feature requests](https://github.com/Atmosphere-NX/Atmosphere/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22features%2Ffeature-request%22) to see if your idea has already been suggested before opening a new issue.]  
```

Reply:
Politeness while ideal, is not the reality. It should be a requirement to reference existing feature requests before a user makes a potentially duplicate issue.

Reply:
Sure, but I think that my suggested changes convey the same idea more clearly and without coming off as harshly. We (or at least I) want to be somewhat inviting. I think the kind of people who see "Note: Please search for previous feature requests" and think it's less mandatory than "Required: Reference the already existing feature request" aren't the kind of people who would do a good job of searching for that anyway.

Reply:
Addressed and rephrased.